library/psa_crypto.c

```diff
@@ -73,7 +73,6 @@
 #include "mbedtls/error.h"
 #include "mbedtls/gcm.h"
 #include "mbedtls/md5.h"
-#include "mbedtls/md.h"
```

Comment:
I think both are debatable, but IMO:
- regarding `psa_crypto.c`: the only thing it uses directly is `mbedtls_md_type_from_psa_alg()` which is declare in `md_psa.h`, and it doesn't use any `MBEDTLS_ERR_MD_xxx` macro (some variables will have those values, but the macros themselves are not used), so I'd say it's OK;
- regarding `rsa.c`: as Valerio pointed out, `md.h` is included from `rsa.h` so I think that's acceptable.

If you agree with this reasoning can you mark the PR as approved for github's benefit? Otherwise feel free to explain why you disagree!